packages/button/src/styles/Button.css

```diff
@@ -400,3 +400,144 @@
   width: 2.4rem;
 }
 /* stylelint-enable no-descending-specificity */
+
+/* ------- PRIMARY FLAIR ------- */
+
+.Button--primaryFlair {
+  background: var(
+    --lp-color-bg-interactive-primary-flair-base-dark,
+    linear-gradient(124deg, #a34fde 0%, #405bff 98.05%)
```

Comment:
There may be potential updates to the color gradient. Antonio and I agreed our current figma designs suffice for now.